training/distributed_training/pytorch/model_parallel/gpt2/submit-train-gpt-simple.ipynb

```diff
@@ -0,0 +1,683 @@
+{
```

Comment:
<p>Grammar/branding - change <code>fsx</code> and <code>FSX</code> to standard spelling <code>FSx</code> in all text/markdown and section titles.</p><p> </p><p>Use page title rather than "here" as link text - <code>Distributed Training of Mask-RCNN in Amazon SageMaker using FSx</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3034/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>